src/binary_to_text/binary_encoder.ml

```diff
@@ -0,0 +1,667 @@
+  | F_binop (S64, Div) -> add_char '\xA3'
+  | F_binop (S64, Min) -> add_char '\xA4'
+  | F_binop (S64, Max) -> add_char '\xA5'
+  | F_binop (S64, Copysign) -> add_char '\xA6'
+  | I32_wrap_i64 -> add_char '\xA7'
+  | I_trunc_f (S32, S32, S) -> add_char '\xA8'
+  | I_trunc_f (S32, S32, U) -> add_char '\xA9'
+  | I_trunc_f (S32, S64, S) -> add_char '\xAA'
+  | I_trunc_f (S32, S64, U) -> add_char '\xAB'
+  | I64_extend_i32 S -> add_char '\xAC'
+  | I64_extend_i32 U -> add_char '\xAD'
+  | I_trunc_f (S64, S32, S) -> add_char '\xAE'
+  | I_trunc_f (S64, S32, U) -> add_char '\xAF'
+  | I_trunc_f (S64, S64, S) -> add_char '\xB0'
+  | I_trunc_f (S64, S64, U) -> add_char '\xB1'
+  | F_convert_i (S32, S32, S) -> add_char '\xB2'
+  | F_convert_i (S32, S32, U) -> add_char '\xB3'
+  | F_convert_i (S32, S64, S) -> add_char '\xB4'
+  | F_convert_i (S32, S64, U) -> add_char '\xB5'
+  | F32_demote_f64 -> add_char '\xB6'
+  | F_convert_i (S64, S32, S) -> add_char '\xB7'
+  | F_convert_i (S64, S32, U) -> add_char '\xB8'
+  | F_convert_i (S64, S64, S) -> add_char '\xB9'
+  | F_convert_i (S64, S64, U) -> add_char '\xBA'
+  | F64_promote_f32 -> add_char '\xBB'
+  | I_reinterpret_f (S32, S32) -> add_char '\xBC'
+  | I_reinterpret_f (S64, S64) -> add_char '\xBD'
+  | F_reinterpret_i (S32, S32) -> add_char '\xBE'
+  | F_reinterpret_i (S64, S64) -> add_char '\xBF'
+  | I_extend8_s S32 -> add_char '\xC0'
+  | I_extend16_s S32 -> add_char '\xC1'
+  | I_extend8_s S64 -> add_char '\xC2'
+  | I_extend16_s S64 -> add_char '\xC3'
+  | I64_extend32_s -> add_char '\xC4'
+  | Ref_null rt ->
+    add_char '\xD0';
+    write_reftype buf rt
+  | Ref_is_null -> add_char '\xD1'
+  | Ref_func idx -> write_char_indice buf '\xD2' idx
+  | I_trunc_sat_f (S32, S32, S) -> write_fc buf 0
+  | I_trunc_sat_f (S32, S32, U) -> write_fc buf 1
+  | I_trunc_sat_f (S32, S64, S) -> write_fc buf 2
+  | I_trunc_sat_f (S32, S64, U) -> write_fc buf 3
+  | I_trunc_sat_f (S64, S32, S) -> write_fc buf 4
+  | I_trunc_sat_f (S64, S32, U) -> write_fc buf 5
+  | I_trunc_sat_f (S64, S64, S) -> write_fc buf 6
+  | I_trunc_sat_f (S64, S64, U) -> write_fc buf 7
+  | Memory_init idx ->
+    write_fc buf 8;
+    write_indice buf idx;
+    add_char '\x00'
+  | Data_drop idx ->
+    write_fc buf 9;
+    write_indice buf idx
+  | Memory_copy ->
+    write_fc buf 10;
+    add_char '\x00';
+    add_char '\x00'
+  | Memory_fill ->
+    write_fc buf 11;
+    add_char '\x00'
+  | Table_init (tableidx, elemidx) ->
+    write_fc buf 12;
+    write_indice buf elemidx;
+    write_indice buf tableidx
+  | Elem_drop idx ->
+    write_fc buf 13;
+    write_indice buf idx
+  | Table_copy (idx1, idx2) ->
+    write_fc buf 14;
+    write_indice buf idx1;
+    write_indice buf idx2
+  | Table_grow idx ->
+    write_fc buf 15;
+    write_indice buf idx
+  | Table_size idx ->
+    write_fc buf 16;
+    write_indice buf idx
+  | Table_fill idx ->
+    write_fc buf 17;
+    write_indice buf idx
+  | _ -> assert false (* TODO *)
```

Comment:
Could you list all the cases here ? So that we don't forget to update it when adding new stuff.